ako-infra/ingestion/avi_infra_controller.go

```diff
@@ -187,6 +189,21 @@ func ConfigureSeGroup(client *clients.AviClient, seGroup *models.ServiceEngineGr
 
 }
 
+func (a *AviControllerInfra) AnnotateSystemNamespace(seGroup string, cloudName string) {
+	nsName := "vmware-system-ako"
+	nsObj, err := a.cs.CoreV1().Namespaces().Get(context.TODO(), nsName, metav1.GetOptions{})
+	if err != nil {
+		utils.AviLog.Fatalf("Failed to update the vmware-system-ako namespace due to the following error :%v", err.Error())
+	}
+	// Update the namespace with the required annotations
+	nsObj.Annotations["ako.vmware.com/wcp-se-group"] = seGroup
+	nsObj.Annotations["ako.vmware.com/wcp-cloud-name"] = cloudName
+	_, err = a.cs.CoreV1().Namespaces().Update(context.TODO(), nsObj, metav1.UpdateOptions{})
+	if err != nil {
+		utils.AviLog.Fatalf("Error occurred while Updating namespace: %v", err)
```

Comment:
I think we can print a warning here, return false and let the caller of SetupSEGroup() make a fatal call if required. It can be same for all false cases. 
Later we may have to add a few retries in case of failure.